saleor/app/manifest_validations.py

```diff
@@ -12,6 +12,7 @@
     get_permissions_enum_list,
     split_permission_codename,
 )
+from ..graphql.webhook.mutations import validate_query
```

Comment:
If you need to import logic from graphql's mutation file then I would say, that the `validate_query` should be placed in a different file. (Previously it was used only in mutations, if we want to use them in second place, let's move it to more common file.)

Reply:
As we decided on a call I'm going to use `validate_subscription_query` (so it will requre it to be non-empty), and there's no reason to move `validate_subscription_query` from `graphql.webhook.subscription_payload`